<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Metro大都会</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/imgs/logo.png</normaloff>:/imgs/logo.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>39</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="contentWidget" native="true">
      <widget class="QWidget" name="mainRightWidget" native="true">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>0</y>
         <width>500</width>
         <height>580</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <widget class="QWidget" name="operatonWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>500</width>
          <height>560</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="startStationLable">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>起始站:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="startStation">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>270</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="startStationEdit">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>50</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="endStationLable">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>110</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>终点站:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="endStation">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>110</y>
           <width>270</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="endStationEdit">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>110</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="priorityLable">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>170</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>优先级:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="priority">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>170</y>
           <width>270</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>时间最短</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>站数最少</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>换乘最少</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="currentQueryButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>210</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
        <widget class="QWidget" name="mapWidget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>260</y>
           <width>500</width>
           <height>300</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/imgs/user.jfif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
